hudi-spark-datasource/hudi-spark3-common/src/main/java/org/apache/hudi/spark3/internal/ReflectUtil.java

```diff
@@ -23,7 +23,7 @@
 import org.apache.spark.sql.catalyst.util.DateFormatter;
 
 import scala.Option;
-import scala.collection.Seq;
+import scala.collection.immutable.Seq;
 import scala.collection.immutable.Map;
```

Comment:
combine into scala.collection.immutable.{Seq, Map} ?

Reply:
These scala imports cannot be combined.  Otherwise the compiler throws error.